argon2/README.md

```diff
@@ -23,17 +23,9 @@ It implements the following three algorithmic variants:
 Support is provided for embedded (i.e. `no_std`) environments, including
 ones without `alloc` support.
 
-## Minimum Supported Rust Version
-
-Rust **1.81** or higher.
-
-Minimum supported Rust version can be changed in the future, but it will be
-done with a minor version bump.
-
 ## SemVer Policy
 
 - All on-by-default features of this library are covered by SemVer
-- MSRV is considered exempt from SemVer as noted above
```

Comment:
We probably can remove this section completely. By default all features are covered by semver unless specified otherwise.